src/extension.ts

```diff
@@ -83,14 +95,27 @@ export function activate(context: ExtensionContext) {
     let disposable = languageClient.start();
     toDispose.push(disposable);
     languageClient.onReady().then(() => {
+
+      // Code Lens actions
+      context.subscriptions.push(commands.registerCommand(Commands.SHOW_XML_REFERENCES, (uriString: string, position: Position) => {
+        const uri = Uri.parse(uriString);
+        workspace.openTextDocument(uri).then(document => {
+          // Consume references service from the XML Language Server
+          let param = languageClient.code2ProtocolConverter.asTextDocumentPositionParams(document, position);
+          languageClient.sendRequest(ReferencesRequest.type, param).then(locations => {
+            commands.executeCommand(Commands.SHOW_REFERENCES, uri, languageClient.protocol2CodeConverter.asPosition(position), locations.map(languageClient.protocol2CodeConverter.asLocation));
```

Comment:
EDITOR_SHOW_REFERENCES or else you end up in an infinite loop

Reply:
Sorry for this big mistake, @fbricon fixed.